bin/src/commands.rs

```diff
@@ -43,8 +42,9 @@ pub async fn add_contract(
 
     let code = provider.get_code_at(address).await?;
     let proxy = ethui_proxy_detect::detect_proxy(address, &provider).await?;
+    let network_is_dev = network.is_dev().await;
 
-    let (name, abi) = if networks_is_dev().await? {
```

Comment:
seems like we can also delete `networks_is_dev` now. don't see any usage of it in the frontend

Reply:
we can, yes, I had that change in another branch